benefits/static/css/admin/theme.css

```diff
@@ -75,6 +75,11 @@ div.breadcrumbs {
   display: none;
 }
 
+#logout-form button:active,
+#logout-form button:hover {
+  margin-bottom: 0 !important; /* Fix logout link hover bug */
```

Comment:
Mind updating this comment to be more specific about the bug, like in your PR description:

> Fixes a bug where the Log out link moves up/down when hovering over it.
> Fixed by applying a `margin-bottom: 0` to links that, the `base.css` (default Django style) adds a `margin-bottom: 1px` to.

Reply:
@thekaveman https://github.com/cal-itp/benefits/pull/2792/commits/35d36394fb64f2e6784a35bac5f3882a502f4764